tensorflow/java/src/main/java/org/tensorflow/Graph.java

```diff
@@ -77,6 +80,22 @@ public Operation operation(String name) {
   }
 
   /**
+   * Returns all operations that exist in the Graph.
+   */
+  public List<Operation> getOperations() {
```

Comment:
Returning as an Iterator opens up an issue with a potential concurrent modification; implementing an assurance (or preferably throwing) against this would be an extra pot of stew. Barring objection, i will note the lack of observance in the javadocs for the ~getter and implement it as spec'd.